databricks-sdk-java/src/main/java/com/databricks/sdk/core/ApiClient.java

```diff
@@ -58,7 +99,26 @@ public ApiClient(DatabricksConfig config) {
   }
 
   public ApiClient(DatabricksConfig config, Timer timer) {
+    this(
+        config,
+        timer,
+        v -> config.authenticate(),
+        v -> config.getHost(),
+        v -> config.getAuthType());
```

Comment:
this(new Builder().withConfig(config.resolve()).withTimer(timer))

Reply:
https://github.com/databricks/databricks-sdk-java/pull/355/commits/652cb0b7c629e2b4a38725ea4069a874352eec88